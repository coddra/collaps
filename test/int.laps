010 .
0xa0 .
0o17 .
0b0110 .
4611686018427387903 + 1 .
