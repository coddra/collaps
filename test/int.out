10
160
15
6
-4611686018427387904
